weblate/templates/addons/squash_help.html

```diff
@@ -4,5 +4,5 @@
 {% trans "All Weblate commits from all components within this repository will be squashed." %}
 </p>
 <p>
-{% trans "The resulting commits will include all commit messages and the commit authorship will not be preserved for all but per author squash. If you want to include authorship information it is recommended to customize the commit message to include this." %}
+{% trans "The resulting commits will include all commit messages, but the commit authorship will be the granted to the one squashing, and not be preserved for all. If you want to include authorship info, it is recommended to customize the commit message to include this." %}
```

Comment:
This is not what it means :-) (what is my fault that the original sentence is hard to understand). There are currently four types of squashing (configurable just above this) and all of them with exception of per author do drop commit authorship information.

See screenshot:

![screenshot_2019-03-03 f-droid f-droid](https://user-images.githubusercontent.com/212189/53693350-975c1700-3d9f-11e9-86a7-53b54d272ab2.png)

Reply:
My bad. My assumption now is that "per author" possibly makes a meta-git user per Weblate user, so that it is Git-specific (git-blame works), and that OTOH adding it to the commit log means it is just historic, unless Weblate has some support for picking it back out?